assets/sass/components/global/_googlesitekit-noscript.scss

```diff
@@ -0,0 +1,50 @@
+/**
+ * Header styles.
+ *
+ * Site Kit by Google, Copyright 2020 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+.googlesitekit-noscript {
+
+	@include googlesitekit-inner-padding;
+	@include shadow;
+
+	background-color: $c-base;
+	margin: 5px 20px 5px 0;
+
+	&__title {
+		margin: 0;
+	}
+}
+
+.no-js {
+
+	#googlesitekit-notice-activated {
+
+		border-left-color: $c-selective-yellow; // noscript renders inside success
+		margin: 5px 15px 5px 0; // fixes the alignment with rest of the page
+		padding: 0;
+
+		.googlesitekit-noscript {
+			border: 0;
+			margin: 0;
+			padding: 0;
+		}
+	}
+
+	[id^="js-googlesitekit-"] {
```

Comment:
What's the aim of this selector? To hide container divs that React would have used to load the React apps? Won't those be empty and thus invisible anyway?

Reply:
No, most/all of them render from the server with loading progress bars which would appear indefinitely otherwise 😄

Reply:
There is a progress bar that get rendered
https://github.com/google/site-kit-wp/pull/1150/files#diff-b6de1dbc42d4247d06044aa0ce40895bR286

Reply:
Ah, could you add a comment then explaining this is intended to target a progress bar then? It just doesn't look that way from context here 😄